MyToolsYourToolsBackend/MyToolsYourToolsBackend.Application/Services/IOpinionService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MyToolsYourToolsBackend.Application.Dtos;
+
+namespace MyToolsYourToolsBackend.Application.Services
+{
+    public interface IOpinionService
+    {
+        IEnumerable<OpinionDto> GetAllOpinions();
+        OpinionDto AddOpinion(OpinionForCreationDto opinion, Guid ratedUserId, Guid ratingUserId);
```

Comment:
Ogólnie do kontrolera dodałbym funkcję typu GetUserOpinions(userId), żeby za każdym razem nie ściągać na front wszystkich opinii i filtrować ich na froncie, tylko przefiltrowane już zwrócić z backendu, ale na razie możemy to zostawić, bo działa dobrze